modules/apps/commerce/commerce-product-definitions-web/src/main/java/com/liferay/commerce/product/definitions/web/internal/frontend/data/set/provider/CommerceProductAccountGroupFDSDataProvider.java

```diff
@@ -64,8 +64,9 @@ public List<CProductAccountGroup> getItems(
 				fdsPagination.getEndPosition(), null);
 
 		for (AccountGroupRel accountGroupRel : accountGroupRels) {
-			AccountGroup accountGroup = _accountGroupService.getAccountGroup(
-				accountGroupRel.getAccountGroupId());
+			AccountGroup accountGroup =
```

Comment:
if this one is used for "eligibility" that it is correct to check for permissions, suppplier should not be allowed to see all other accounts/accountGroups

Reply:
visibility is more or less like eligibility, so they should not be able to see other account / accountGroups